Modules/ThirdParty/VNL/README.md

```diff
@@ -1,7 +1,9 @@
-The vxl subdirectory contains a portion of the vxl project from
-
+The `VNL` module contains a portion of the `VXL` project from
 
   https://github.com/vxl/vxl.git
-  Documentation: http://vxl.sourceforge.net
 
-See `./src/README-ITK.txt` for details and instructions to update from upstream.
+Documentation about the library can be found at
+
+  http://vxl.sourceforge.net
```

Comment:
This is now: https://vxl.github.io/

Reply:
That's one of the changes I intended to do, but looks like I left it behind... Thanks for the review Matt. Fixed in 0308920.